combat-trainer.lic

```diff
@@ -222,9 +224,6 @@ class LootProcess
     @necro_heal = thanatology['heal'] || false
     echo("  @necro_heal: #{@necro_heal}") if $debug_mode_ct
 
-    @necro_train_first_aid = thanatology['train_first_aid'] || false
```

Comment:
If you are removing this setting, you'll probably want to add a validator that checks if it is `true` and warns the user they need to change their settings.